taichi/codegen/spirv/kernel_utils.h

```diff
@@ -262,6 +278,9 @@ class KernelContextAttributes {
   size_t args_bytes_{0};
   size_t rets_bytes_{0};
   size_t extra_args_bytes_{0};
+
+  const lang::StructType *args_type_;
```

Comment:
nit: since we have a default constructor and this is marked const, initialize it with `{nullptr}`